README.md

```diff
@@ -1,24 +1,92 @@
-# README
+# Project: Private Events
```

Comment:
- [ ] As requested earlier on, add instructions on how to run the tests in this file.
- [ ] [OPTIONAL] Since the previous reviewer didn't configure tailwind as expected, add instructions on how to set up tailwind to have the same views.

Reply:
make sure you have the latest version of postcss and webpack installed
npm install postcss@latest autoprefixer@latest
bundle exec rails webpacker:install

This how I did the Tailwindcss setup

1: yarn add tailwindcss

2: yarn tailwind init
this will generate the tailwindcss.config.js file in the root of the project
which i will later moved to the app/javascript/stylesheets folder

4: In the above mentioned dir create the file  application.scss and put these lines in it.
`@tailwind base;

@tailwind components;

@tailwind utilities;`

5: In app/javascript/packs/application.scss file put this line:
    import "stylesheets/application"

6: Add these lines to postcss.config.js file in the root dir of the project
   var tailwindcss = require('tailwindcss');
    plugins: [
    tailwindcss('./app/javascript/stylesheets/tailwind.config.js'),
    . . . . .
    . . . . .
    ] (you can check this file in my project)